ui/changes/src/index.html

```diff
@@ -58,6 +58,10 @@ <h3>Filters</h3>
         <label>White Board
           <input type="text" name="whiteboard" id="whiteBoard">
         </label>
+        <label>
+          Version<br />
+          <select name="releaseVersion" id=releaseVersion></select>
```

Comment:
I think we might want to make this a <select multiple> and then handle checking across multiple releases in the filtering code. That way by default we'll continue to show everything that matches other filters (i.e. in the date range) and not also require changing to the appropriate release.

We do this for others like testing tags etc so there are some examples of this in buildTable.

Reply:
Make this a what?

Reply:
Sorry, I don't know what happened with that comment. Add the `multiple` attribute to this select tag i.e. `<select name="releaseVersion" id="releaseVersion" multiple>`

Reply:
You can take a look at testing tags, teams and components for examples of what @bgrins is suggesting